jfr-streaming/src/main/java/io/opentelemetry/contrib/jfr/Constants.java

```diff
@@ -0,0 +1,10 @@
+package io.opentelemetry.contrib.jfr;
+
+public class Constants {
+    private Constants() {}
+
+    public static final String THREAD_NAME = "thread.name";
```

Comment:
Use an `AttributeKey` for the constant rather than creating one every time

Reply:
Fixed.